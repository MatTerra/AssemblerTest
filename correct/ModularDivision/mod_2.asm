begin
section data
n: space
section text
mod_2:
finish: extern
mod_1: extern
divide: extern
public mod_2
public n
load n
jmpz finish
jmp divide
end
