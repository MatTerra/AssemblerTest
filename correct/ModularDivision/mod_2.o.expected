H: mod_2
H: 7
H: 0101010
H: U divide 5
H: U finish 3
H: D mod_2 0
H: D n 6
T: 10 06 08 00 05 00 0
